sagemaker-python-sdk/tensorflow_script_mode_using_shell_commands/tensorflow_script_mode_using_shell_commands.ipynb

```diff
@@ -0,0 +1,303 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# TensorFlow Script Mode - Using Shell commands\n",
+    "\n",
+    "Starting by TensorFlow's framework version 1.11, you can shell commands as\n",
+    "your training entry point. Shell scripts are useful for many use cases including:\n",
+    "\n",
+    "- Invoke Python scripts with specific parameters\n",
+    "- Install requirements before training\n",
```

Comment:
I thought the general sentiment was opposed to advertising run-time installation of dependencies.